requirements.txt

```diff
@@ -1,31 +1,31 @@
 # #############################################################################
 # requirements
 # #############################################################################
-pywin32 >= 303; python_version > '2.7'
+pywin32==303; python_version > '2.7'
 # https://github.com/mhammond/pywin32
 # License: PSF
 
-pythonnet >= 2.4.0; python_version > '2.7'
+pythonnet==2.5.2; python_version > '2.7'
 # https://github.com/pythonnet/pythonnet
 # License: MIT
 
-rpyc >= 5.0.1; python_version > '2.7'
+rpyc==5.1.0; python_version > '2.7'
```

Comment:
I would need to check if this version is compatible with the one embedded in pyaedt for ironpython usage. Should be ok but it good to have a try on linux machine next week before closing it or simply stay with 5.0.1 for now.

Reply:
```suggestion
rpyc==5.0.1; python_version > '2.7'
```